WTWSMS/history/characters/greek.txt

```diff
@@ -3625,4 +3625,31 @@
 	434.1.1={
 		death=yes
 	}
-}
\ No newline at end of file
+}
+1310010= {
+	name="Alexander"
```

Comment:
Alexandros in Greek, no?

Reply:
He is named Alexander in the PLRE, I think I named "Alexandros" the dynasty

Reply:
Yes, but the PLRE is English, and for French and Spanish users we can't use Anglicised ruler names.

Reply:
Ok, I'll change it, what would the dynasty name be?

Reply:
No idea, I'm not enough familiar with Greek. How have the other Greek dynasty names been done?

Reply:
All I've seen end with -as or -os so then the name should be Alexandros Alexandros / Alexandros Alexandrid /Alexandros Alexander? That are the best 3 possibilities I see

Reply:
"id" is meant to be a steppe-suffix as far as I know. "ites" seems to be the Greek one in the culture file, so I would advice Alexandros Alexandrites based upon that.

Reply:
Ok, Agreed